GeoLib/AnalyticalGeometry.h

```diff
@@ -52,6 +53,17 @@ Orientation getOrientation (const GeoLib::Point* p0,
                             const GeoLib::Point* p1,
                             const GeoLib::Point* p2);
 
+
+/**
+ * Tests if p is right of the line defined by a and b in 2D.
+ * (Note: This functionality is similar to getOrientation(). It is probably
+ * not quite as exact in numerically challenging situations but requires only
+ * about half the runtime.)
```

Comment:
I'd recommend to prefer the correctness over runtime in such cases.

Reply:
This is probably my computer graphics POV but nothing bad happens if this gives a result that is wrong by a measure of epsilon outside of the elevation also being wrong by an epsilon (which is most likely less of an error than any rounding artifacts, measurement errors, etc that happened before already). Should I still remove it and use the slower method?

Reply:
@TomFischer your decision.

Reply:
I would take the safe implementation.